modules/git/repo_branch_nogogit.go

```diff
@@ -59,10 +59,10 @@ func (repo *Repository) IsBranchExist(name string) bool {
 	return repo.IsReferenceExist(BranchPrefix + name)
 }
 
-// GetBranchNames returns branches from the repository, skipping skip initial branches and
+// GetBranchNames returns branches from the repository, skipping initial branches and
```

Comment:
The skip in this line referred to the argument `skip`. If you want to remove it you should change the next line too.

I.e. 
```
// GetBranchNames returns branches from the repository, limited by the provided arguments or all branches if the provided limit is 0
```

Reply:
Oh I see, I think I can update the comments with "quotes", then it should be clear

Reply:
Updated with quotes (to avoid spellcheckers reporting spelling errors).

If the quote is not liked, I can revert these changes with no quoting (as before).